### ISSUE TITLE
Upgrade broke the docker installation

### ISSUE THREAD

**Issue by valerykustov** [pgadmin-org/pgadmin4#5898]
Hello!

I have a pgadmin4 server running in docker. The docker-compose.yml is the following:
```
version: '3'

services:
  pgadmin:
    image: dpage/pgadmin4:6.18
    environment:
      PGADMIN_DEFAULT_EMAIL: my_email
      PGADMIN_DEFAULT_PASSWORD: my_password
    volumes:
       - ./pgadmin_data:/var/lib/pgadmin
       - ./config_local.py:/pgadmin4/config_local.py
    ports:
      - "some_ip:5050:80"
    restart: unless-stopped
```

I've tried to upgrade the installation with replacing the `image` key in this file with `dpage/pgadmin4:6.20` and running `docker-compose down && docker-compose up -d`. The downloading of the newer image went well. However it didn't start. `docker-compose logs` are attached in 6.20.log. After that I've tried to upgrade to 6.19 at least, but it failed, too. And now I can not revert to 6.18 neither. Both logs for 6.19 and 6.18 are attached with respective names, too. Though it seems to me that all of the logs are about the same problem with a migration.

How do I fix that without reinstalling the pgAdmin4 completely? Any advice is appreciated!

Logs:
[6.20.log](https://github.com/pgadmin-org/pgadmin4/files/10834594/6.20.log)
[6.19.log](https://github.com/pgadmin-org/pgadmin4/files/10834595/6.19.log)
[6.18.log](https://github.com/pgadmin-org/pgadmin4/files/10834596/6.18.log)

**Comment by yogeshmahajan-1903** [comment]
@valerykustov 
From which pgAdmin 4 version, you have tried to upgrade to 6.20?
In you volume mount, db backup of your old version is preserved with 'pgadmin4.db.prev.bak' file name. 
You can copy this file to current pgadmin4.db file & try to launch pgadmin with your older version.

**Comment by valerykustov** [comment]
@yogeshmahajan-1903 Thank you for your reply! Copying the db file helped and pgadmin started. The version I was trying to upgrade from is 6.18. Now this version is running OK.

**Comment by akshay-joshi** [comment]
@valerykustov 

I am not able to reproduce the issue. Steps that I tried:

1. Create a "pgadmin-docker.yaml" file same as mentioned in the issue. with 6.18 version
2. Run docker-compose -f "pgadmin-docker.yaml" up -d to start the container.
3. Created 3 users and added one shared server from the admin user. 
4. Log in with all 3 users one by one.
5. Change the pgadmin version to 6.20 in the "pgadmin-docker.yaml".
6. Run docker-compose -f "pgadmin-docker.yaml" down
7. Run docker-compose -f "pgadmin-docker.yaml" up -d

pgAdmin 4 v6.20 upgraded properly and working fine. Refer to the image below:
<img width="1199" alt="Docker" src="https://user-images.githubusercontent.com/1632882/222729214-25085013-e8dd-4949-a417-f68130cbba81.png">

Can you please help us to reproduce this issue? 
Will it be possible for you to share your pgadmin4.db file? If not possible we need to identify the exact cause.
Have you upgraded to the 6.18 or it's a fresh image that has been created?

**Comment by valerykustov** [comment]
https://github.com/akshay-joshi

I may share the DB but It seems that it has sensitive information. Will it be convenient if I provide the pgadmin4.db file with following tables truncated: `database`, `keys`, `query_history`, `server`, `user`? Maybe there are other tables I should truncate before sharing the db file?

It definitely was upgraded before but unfortunately I cannot remember what version was before 6.18. Probably this installation was running even an older major version some time ago.

**Comment by akshay-joshi** [comment]
@valerykustov 

We are able to reproduce the issue when you upgrade directly from v6.2 and below to v6.16 and above then the issue is reproducible. We have fixed the issue and it will be available in the next release v6.21 on coming Thursday.

I am closing this issue as it is a Duplicate of #5870
